spec/protocol/addresses_keys/addresses.md

```diff
@@ -19,3 +19,8 @@ $pk_d = [ivk]B_d$
 The *compact flag key* is derived from a (diversified) *compact detection key* $cdtk_d$ using a non-diversified `decaf377` basepoint [$B$](../primitives/decaf377/test_vectors.md):
 
 $cfk_d = [cdtk_d]B$
+
+The diversifier, transmission key, and compact flag key are [Bech32m](https://github.com/bitcoin/bips/blob/master/bip-0350.mediawiki) encoded with a human readable prefix that is:
```

Comment:
while this is not the nomenclature that we're currently using for FMD, I kept this as is to avoid inconsistency/confusion until #81